maxim/evaluators/base_evaluator.py

```
@@ -32,13 +32,13 @@ def pass_fail_criteria(self):
 
     @abstractmethod
     def evaluate(
-        self, result: LocalEvaluatorResultParameter, data: ManualData
+        self, result: LocalEvaluatorResultParameter, data: LocalData
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)